lib/plugins/digging.js

```diff
@@ -242,7 +242,7 @@ function inject (bot) {
     return block.digTime(
       type,
       creative,
-      bot.entity.isInWater,
+      bot.blockAt(bot.entity.position.offset(0, bot.entity.eyeHeight, 0)).name === 'water',
```

Comment:
I think this would be better if you put it into a function for clarity

Eg
```js
bot._getBlockAtEyeLevel = () => bot.entity.position && bot.blockAt(bot.entity.position.offset(0, bot.entity.eyeHeight, 0))
```